torch_xla/csrc/lowering_context.cpp

```diff
@@ -2,66 +2,72 @@
 
 #include <torch/csrc/lazy/core/ir_metadata.h>
 
-#include <iostream>
 #include <optional>
 #include <sstream>
 #include <stdexcept>
-#include <string_view>
+#include <utility>
 
+#include "absl/log/absl_check.h"
+#include "absl/log/absl_log.h"
 #include "absl/status/status.h"
 #include "absl/strings/str_cat.h"
-#include "absl/strings/str_join.h"
 #include "absl/strings/str_replace.h"
-#include "absl/strings/string_view.h"
 #include "torch_xla/csrc/ir.h"
 #include "torch_xla/csrc/runtime/computation_client.h"
 #include "torch_xla/csrc/runtime/debug_macros.h"
 #include "torch_xla/csrc/runtime/sys_util.h"
 #include "torch_xla/csrc/shape_helper.h"
 #include "torch_xla/csrc/stack_frame_index_builder.h"
-#include "torch_xla/csrc/unwrap_data.h"
 
 namespace torch_xla {
 
 namespace {
 
 class HloMetadataSetter {
  public:
-  HloMetadataSetter(LoweringContext* loctx, const torch::lazy::Node* node) {
+  HloMetadataSetter(LoweringContext* const low_ctx,
```

Comment:
Since we are at it, why don't we make this also a reference: `LoweringContext& low_ctx`? And, maybe, call it `lowering_context`?